decidim-core/app/helpers/decidim/resource_helper.rb

```diff
@@ -0,0 +1,64 @@
+# frozen_string_literal: true
+module Decidim
+  # A Helper to render and link to resources.
+  module ResourceHelper
+    # Builds the path to a resource. Useful when linking to a resource from
+    # another engine.
+    #
+    # resource - An object that is a valid resource exposed by some feature.
+    #
+    # Returns a String.
+    def decidim_resource_path(resource)
+      _decidim_resource_route(resource, "path")
```

Comment:
I think we need to decide a style for this so everyone uses `_` for private methods or not.